utils/utils.go

```diff
@@ -543,19 +541,6 @@ func ConvertYAMLtoJSON(yamlData []byte) ([]byte, error) {
 	return jsonData, nil
 }
 
-// ConvertYAMLtoJSONPretty will do exactly what you think it will. It will deserialize YAML into serialized JSON.
-// However, this version will apply prefix/indentation to the JSON.
-func ConvertYAMLtoJSONPretty(yamlData []byte, prefix string, indent string) ([]byte, error) {
```

Comment:
Why was this removed?

Reply:
This was replaced by the Ordered Json here: https://github.com/pb33f/libopenapi/pull/205/files#r1425740867